src/main/java/org/jabref/gui/preferences/table/TableTabViewModel.java

```diff
@@ -123,7 +123,6 @@ public void setValues() {
 
         availableColumnsProperty.clear();
         availableColumnsProperty.addAll(
-                new MainTableColumnModel(MainTableColumnModel.Type.MATCH_SCORE),
```

Comment:
Its a bit a pity that we loose this uncertain matches. But OK to loose if we get matches at all 🤣🤣. -- Maybe also worth noting in the report and/or blog post.